Client/Helpers/FxALoginHelper.swift

```diff
@@ -155,6 +155,12 @@ class FxALoginHelper {
             account.updateProfile()
         }
         
+        if AppConstants.MOZ_ENABLE_LEANPLUM && AppConstants.MOZ_FXA_LEANPLUM_AB_PUSH_TEST {
```

Comment:
Is `MOZ_FXA_LEANPLUM_AB_PUSH_TEST` ever going to be `true` when `MOZ_ENABLE_LEANPLUM` is `false`? Remove `MOZ_ENABLE_LEANPLUM` test.

Reply:
It shouldn't be the case, but this was added to guard against the possibility of a bad configuration.